apps/jsonrpc2/lib/jsonrpc2/request.ex

```diff
@@ -0,0 +1,41 @@
+defmodule JSONRPC2.Request do
+  @moduledoc """
+  JSON-RPC 2.0 Request object utilites.
+  """
+
+  @type request ::
+          {JSONRPC2.method(), JSONRPC2.params()}
+          | {JSONRPC2.method(), JSONRPC2.params(), JSONRPC2.id()}
+
+  @doc """
+  Returns a serialized `request` using `serializer`.
+
+  See the README for more information on serializers.
```

Comment:
These could use `Examples` which would help clear up exactly how the function works.